src/main/java/io/securecodebox/persistence/defectdojo/service/GenericDefectDojoService.java

```diff
@@ -73,14 +75,19 @@ public final T get(long id) {
 
     final var url = this.config.getUrl() + API_PREFIX + this.getUrlPath() + "/" + id;
     log.debug("Requesting URL: {}", url);
-    ResponseEntity<T> response = restTemplate.exchange(
-      url,
-      HttpMethod.GET,
-      payload,
-      getModelClass()
-    );
-
-    return response.getBody();
+    try {
+      ResponseEntity<T> response = restTemplate.exchange(
+        url,
+        HttpMethod.GET,
+        payload,
+        getModelClass()
+      );
+
+      return response.getBody();
+    } catch (RestClientException e) {
+      log.error("Exception while getting data: {}", e.getMessage());
```

Comment:
Whats in a message for a request client exception?
Is that enough for users to understand what request failed? Does it include the http method and url of the request?

Reply:
That's a common pattern in Java that you simply log the exception message, if the exception is rethrown with the origin exception as cause because the full exception handling is done where the rethrown exception is catched. It's quite debatable if this log message is really necessary.